tracer/tracer_test.go

```diff
@@ -282,6 +282,28 @@ func TestTracerServicesDisabled(t *testing.T) {
 	assert.Equal(0, len(transport.services))
 }
 
+func TestTracerMeta(t *testing.T) {
```

Comment:
One case to test: when we set a meta at the span-level which has the same key as one set as the tracer-level.

And to test something closer to user-land, we should do asserts on spans only (and don't need getAllMeta)